README.md

```diff
@@ -82,8 +82,11 @@ You can also follow the recent developments of the project in the OpenZeppelin [
 [Twitter account](https://twitter.com/openzeppelin).
 
 ## Contribute
+OpenZeppelin exists thanks to its contributors. There are many ways you can participate and help build high quality software. 
```

Comment:
I'm not sure if the "there are many ways you can contribute" still makes sense after moving the links to the community accounts to another section. WDYT about changing the entire section to sth like...?

> To set up a local development environment, clone the repository and run `yarn` in the root of the project. Please refer to the contribution guide for more info on contributing to the OpenZeppelin SDK.